src/main/java/org/altadoon/gt6x/features/verticalmixers/VerticalMixers.java

```diff
@@ -0,0 +1,86 @@
+        mat = MT.DATA.Kinetic_T[4]; MTEx.gt6xMTEReg.add("Vertical Mixer ("                         +mat.getLocal()+")", "Basic Machines"                      , 110, 20001, aClass, mat.mToolQuality, 16, MTEx.MachineBlock     , UT.NBT.make(NBT_MATERIAL, mat, NBT_HARDNESS,   7.0F, NBT_RESISTANCE,   7.0F, NBT_INPUT,   2048, NBT_TEXTURE, "mixer_vertical", NBT_ENERGY_ACCEPTED, TD.Energy.RU, NBT_RECIPEMAP, RM.Mixer, NBT_INV_SIDE_IN, SBIT_U|SBIT_L, NBT_INV_SIDE_AUTO_IN, SIDE_LEFT, NBT_INV_SIDE_OUT, SBIT_D|SBIT_R, NBT_INV_SIDE_AUTO_OUT, SIDE_RIGHT, NBT_TANK_SIDE_IN, SBIT_U|SBIT_R, NBT_TANK_SIDE_AUTO_IN, SIDE_TOP, NBT_TANK_SIDE_OUT, SBIT_D|SBIT_L, NBT_TANK_SIDE_AUTO_OUT, SIDE_BOTTOM, NBT_ENERGY_ACCEPTED_SIDES, SBIT_B, NBT_PARALLEL, 32, NBT_PARALLEL_DURATION, T));
+
+        CR.shapeless(MTEx.gt6xMTEReg.getItem(107),new Object[]{MTEx.gt6Registry.getItem(20181), OreDictToolNames.wrench});
+        CR.shapeless(MTEx.gt6xMTEReg.getItem(108),new Object[]{MTEx.gt6Registry.getItem(20182), OreDictToolNames.wrench});
+        CR.shapeless(MTEx.gt6xMTEReg.getItem(109),new Object[]{MTEx.gt6Registry.getItem(20183), OreDictToolNames.wrench});
+        CR.shapeless(MTEx.gt6xMTEReg.getItem(110),new Object[]{MTEx.gt6Registry.getItem(20184), OreDictToolNames.wrench});
+
+        CR.shapeless(MTEx.gt6Registry.getItem(20181),new Object[]{MTEx.gt6xMTEReg.getItem(107), OreDictToolNames.wrench});
+        CR.shapeless(MTEx.gt6Registry.getItem(20182),new Object[]{MTEx.gt6xMTEReg.getItem(108), OreDictToolNames.wrench});
+        CR.shapeless(MTEx.gt6Registry.getItem(20183),new Object[]{MTEx.gt6xMTEReg.getItem(109), OreDictToolNames.wrench});
+        CR.shapeless(MTEx.gt6Registry.getItem(20184),new Object[]{MTEx.gt6xMTEReg.getItem(110), OreDictToolNames.wrench});
+
+        /* EU */
+        aClass = MultiTileEntityBasicMachineElectric.class;
+        mat = MT.DATA.Electric_T[1]; MTEx.gt6xMTEReg.add("Vertical Electric Mixer ("                          +VN[1]+")", "Basic Machines"                      , 111, 20001, aClass, mat.mToolQuality, 16, MTEx.MachineBlock     , UT.NBT.make(NBT_MATERIAL, mat, NBT_HARDNESS,   4.0F, NBT_RESISTANCE,   4.0F, NBT_INPUT,   32, NBT_TEXTURE, "electricmixer_vertical", NBT_ENERGY_ACCEPTED, TD.Energy.EU, NBT_RECIPEMAP, RM.Mixer, NBT_EFFICIENCY, 5000, NBT_INV_SIDE_IN, SBIT_U|SBIT_L, NBT_INV_SIDE_AUTO_IN, SIDE_LEFT, NBT_INV_SIDE_OUT, SBIT_D|SBIT_R, NBT_INV_SIDE_AUTO_OUT, SIDE_RIGHT, NBT_TANK_SIDE_IN, SBIT_U|SBIT_R, NBT_TANK_SIDE_AUTO_IN, SIDE_TOP, NBT_TANK_SIDE_OUT, SBIT_D|SBIT_L, NBT_TANK_SIDE_AUTO_OUT, SIDE_BOTTOM, NBT_ENERGY_ACCEPTED_SIDES, SBIT_B, NBT_PARALLEL, 4, NBT_PARALLEL_DURATION, T));
```

Comment:
I would call this "Electric Mixer (Vertical, "+mat.getLocal()+")" instead, same for the kinetic one: "Mixer (Vertical, Bronze)" etc.

Reply:
Agreed to use the voltage instaed of mat, but in that case call it "Electric Mixer (LV, Vertical)" etc. instead of "Vertical Electric Mixer (LV)" just to make it clear it is the same machine without any recipe or power changes. Otherwise users could think it might be something else that mixes stuff vertically or something stupid.

Reply:
fixed in [464f981](https://github.com/BorisMulder/gt6x/pull/13/commits/464f9810c84aaf979740f13a7817cb2023daec95)